homeassistant/components/overkiz/sensor.py

```diff
@@ -402,6 +403,9 @@ def native_value(self) -> StateType:
         if self.entity_description.native_value:
             return self.entity_description.native_value(state.value)
 
+        if isinstance(state.value, (dict, list)):
```

Comment:
This was not planned for this PR, but the typing is broken in `dev` already. Since I moved the typing in this PR, I fixed this bug to become 100% mypy compliant. Perhaps a rule did change after the last PR.

```
homeassistant/components/overkiz/sensor.py:405: error: Incompatible return value type (got "Union[str, float, Dict[Any, Any], List[Any]]", expected "Union[None, str, int, float]")  [return-value]
```